src/js/components/Box.js

```diff
@@ -62,6 +62,7 @@ export default class Box extends Component {
     this._addPropertyClass(classes, CLASS_ROOT, 'separator');
     this._addPropertyClass(classes, CLASS_ROOT, 'size');
     this._addPropertyClass(classes, CLASS_ROOT, 'textAlign', 'text-align');
+    this._addPropertyClass(classes, CLASS_ROOT, 'width');
```

Comment:
Can you elaborate on why we would want `initial` as a size choice? From my reading of this property it sounds like it applies to inherited properties, but `width` isn't, is it?

For similar reasons, I'm wondering why we'd want to have `inherit` be a possible width value?

I believe `auto` is the same behavior as not specifying a width/size.